.github/workflows/CI.yml

```diff
@@ -69,13 +68,53 @@ jobs:
         uses: actions/setup-java@v1
         with:
           java-version: 1.8
-      - name: Run UATs
+      - name: Install CLI and dependencies
         run: |
-          coverage run --source=gdk -m pytest -v -s uat
+          pip install .
+          gdk --help
+          pip install -r test-requirements.txt
```

Comment:
Related to previous commit, L56 installs test requirements even before testing CLI tool verification.

Reply:
When customers use pip command to install the cli tool, test-requirements are not installed instead the dependencies in the `requirements` file are only installed as part of setup script. So, we want to replicate the same behavior.